feat(card): added flat variation (#2886)
patternfly/patternfly: src/patternfly/components/Card/examples/Card.md

| `.pf-m-hoverable` | `.pf-c-card` | Modifies the card to include hover styles on `:hover`. |
| `.pf-m-selectable` | `.pf-c-card` | Modifies a selectable card so that it is selectable. |
| `.pf-m-selected` | `.pf-c-card.pf-m-selectable` | Modifies a selectable card for the selected state. |

> | `.pf-m-border` | `.pf-c-card` | Modifies the card to have a border instead of a shadow. `.pf-m-border` is for use in layouts where cards are against a white background.

Reviewer: ```suggestion
| `.pf-m-border` | `.pf-c-card` | Modifies the card to have a border and removes the default shadow. Use this modifier in layouts where cards are against a white background.
```
